marimo/_config/config.py

```diff
@@ -133,6 +141,9 @@ class RuntimeConfig(TypedDict):
     - `default_sql_output`: the default output format for SQL queries. Can be one of:
         `"auto"`, `"native"`, `"polars"`, `"lazy-polars"`, or `"pandas"`.
         The default is `"auto"`.
+    - `default_auto_download`: an Optional list of export types to automatically:
+       `html`, `markdown`, `ipynb`.
```

Comment:
```suggestion
       `html`, `ipynb`.
```

I suggest we remove markdown because markdown doesn't have outputs (it's also not an option in the frontend UI). It's cheap to export as markdown from the CLI (but not cheap to go to HTML or IPYNB since that runs the notebook)

Reply:
we alreayd support this in `auto_download`, so i think it makes sense for parity (we don't need to expose it in the UI though)